added new device + enhanced setup steps

domdorn opened the pull request:
- added DS1819+ to supported list. 
- added steps to verify working installation
- create folders required to store artifacts + toolkit_tarballs
- added toolkit_tarballs volume mount to docker command

domdorn committed:
added new device + enhanced setup steps

- added DS1819+ to supported list. 
- added steps to verify working installation
- create folders required to store artifacts + toolkit_tarballs
- added toolkit_tarballs volume mount to docker command

runfalk commented on README.rst:
```diff
@@ -190,6 +192,34 @@ them from here: https://sourceforge.net/projects/dsgpl/files/toolkit/DSM<DSM_VER
 
 
+Testing the installation
+------------------------
```
This is great. Could you add a sentence that explicitly states that these commands should be run on the NAS?

domdorn replied:
Done

runfalk commented on README.rst:
```diff
@@ -52,6 +52,7 @@ DS1511+   x64        6.2         Yes
 DS1618+   denverton  6.2         Yes
 DS1817+   avoton     6.2         Yes
 DS1815+   avoton     6.2         Yes
+DS1819+   denverton  6.2         Yes (starting with WireGuard-1.0.20210424)
```
Did it not work for older versions or did you not try it?

domdorn replied:
I tried it last time last summer and it did not work back then... package and kernel module compiled fine, but actual networking didn't. unfortunately, I didn't save the old version number before deploying the new package, so can't say for sure which version it was back then. 
I could remove the "starting .." text, as anyway someone now coming to the page will try with the newest version.

runfalk replied:
Yeah, remove that and I'll merge this :+1:

runfalk reviewed (changes requested):
Thanks for the additions!

domdorn committed:
adjusted README

- removed note about Wireguard version for DS1819+
- clarified testing procedure
- explained toolkits_tarball volume enhancement
